README.md

```diff
@@ -273,3 +273,24 @@ following ways, in precedence order:
   in the case where you specify a path to a Python script via the `APP_FILE` environment
   variable, defaulting to a file named `app.py` if it exists. The script is
   passed to a regular Python interpreter to launch your application.
+
+Hot deploy
+---------------------
+
+If you are using Django, hot deploy will work out of the box.
```

Comment:
No and no, but this is only applicable to running with Django, not Gunicorn. Django is only intended to run as a development web server, see: https://github.com/openshift/sti-python/blob/master/3.4/s2i/bin/run#L48 and https://docs.djangoproject.com/en/1.8/ref/django-admin/#runserver-port-or-address-port